### PR TITLE
[Yiran]

### PR DESCRIPTION
**Pull request by yrong07**

### CODE REVIEW

**Review by CarterCChen — APPROVED** [review]
lets go

File: src/test/java/utils/Utilities.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 public class Utilities {
 
+	public static final String name="Yiran Rong";
+
```
Comment (shuqin32) [inline src/test/java/utils/Utilities.java:18]: good to go

**Comment by shuqin32** [comment]
good to go

**Review by shuqin32 — APPROVED** [review]

**Review by tylee17 — APPROVED** [review]

**Review by aspchen805 — APPROVED** [review]
LGTM!

**Review by Muradil-Erkin — APPROVED** [review]

**Pull request merged**